build.sbt

```diff
@@ -3,7 +3,7 @@ import com.typesafe.sbt.SbtGit.git
 
 moduleName := "tofu"
 
-val libVersion = "0.9.1"
+val libVersion = "0.9.3"
```

Comment:
Why?

Reply:
Because it is a bugfix.
Semver tells to update minor version on bugfixes AFAIR

Reply:
It's not you (or another contributor) who bumps the version. Please remove it and leave to those who prepare the release

Reply:
Got it.
```suggestion
val libVersion = "0.9.1"
```